.github/workflows/nightly.yml

```diff
@@ -27,4 +27,10 @@ jobs:
     name: Polybft Property Tests
     uses: ./.github/workflows/property-polybft.yml
     needs: build
+
+  fuzz:
+    name: Fuzz Tests
+    uses: ./.github/workflows/fuzz-test.yml
+    needs: build
```

Comment:
Should it dependant upon `build` workflow? Seems to me that we don't need it.

Reply:
Discussed it privately, and @Nemanja0x left it because if a build fails, we shouldn't trigger tests. Though it is not explicitly needed for unit tests, fuzz tests per se.